cfgov/unprocessed/apps/retirement/css/claiming.less

```diff
@@ -259,12 +233,7 @@
     padding: 0;
   }
 
-  .compared-to-full {
```

Comment:
This element is hidden initially and then shows up when you have the graph showing and then move the range slider. This appears to happen via JS, so I don't see how this CSS would have been used. But please scrutinize if this removal is fine.